m3-transport-mapping-mp/src/main/kotlin/com/finyou/fintrack/backend/mapping/mp/toTransport.kt

```diff
@@ -0,0 +1,119 @@
+package com.finyou.fintrack.backend.mapping.mp
+
+import com.finyou.fintrack.backend.common.context.FtContext
+import com.finyou.fintrack.backend.common.models.*
+import com.finyou.fintrack.backend.common.utils.Const
+import com.finyou.fintrack.kmp.models.*
+import java.util.*
+
+fun FtContext.toInitResponse() = InitTransactionResponse(
+    requestId = onRequest,
+    errors = errors.transport,
+    result = if (errors.isNotEmpty()) InitTransactionResponse.Result.ERROR
+    else InitTransactionResponse.Result.SUCCESS
+)
+
+fun FtContext.toCreateResponse() = CreateTransactionResponse(
+    requestId = onRequest,
+    createdTransaction = responseTransaction.transport,
```

Comment:
Транспортные модели всегда Nullable. Внутренние модели в подавляющем большинстве случаев Non-Nullable. Здесь должна быть обработка случае, когда нужно отдать Null.
```suggestion
    createdTransaction = responseTransaction.transport.takeIf { it != NONE },
```